tests/manual/all-features.md

```diff
@@ -42,6 +42,8 @@ The "previews in view" mode is enabled, which means that previews should be visi
 
 **Text part language** - There are 2 paragraphs with applied language: Spanish with left-to-right direction and Arabic with right-to-left direction. Both should have italic style. Play with the selection and the language dropdown.
 
+**HTML comments** - There are 10 comments in this section: from `C1` to `C10`. Open the source editing mode (or just call the `editor.getData()`) and verify that they are present. Open CKEditor 5 inspector and verify, that there are 10 $root attributes, one for each comment.
```

Comment:
Same here, I'd remove the section about root attributes.

Reply:
Done.